web-frontend/src/pages/TransferCoins/SendCoins/ContactList.tsx

```diff
@@ -0,0 +1,54 @@
+import {
+  MassaToken,
+  PopupModal,
+  PopupModalContent,
+  PopupModalHeader,
+  Selector,
+  Identicon,
+} from '@massalabs/react-ui-kit';
+import { formatStandard } from '../../../utils/MassaFormating';
+import { useResource } from '../../../custom/api';
+import { AccountObject } from '../../../models/AccountModel';
+
+function AccountSelect({ ...props }) {
+  const { modalAccounts, setModalAccounts, setRecipient, account } = props;
+  const { data: accounts = [] } = useResource<AccountObject[]>('accounts');
+  const otherAccounts = accounts.filter(
+    (acnt: AccountObject) => acnt.nickname !== account.nickname,
+  );
```

Comment:
Waht kind of filter we are trying here!? 🤔 What we want to filter!?

Reply:
this is all the accounts but the active one ( so that the sender is different from the recipient)

Reply:
this is when we do a transfer to another of our account

Reply:
WDYT to be a bit more instructive like:

```suggestion
  const { modalAccounts, setModalAccounts, setRecipient, account as selectedAccount } = props;
  const { data: accounts = [] } = useResource<AccountObject[]>('accounts');
  const filteredAccounts = accounts.filter(
    (account: AccountObject) => account.nickname !== selectedAccount.nickname,
  );
```